torch/csrc/autograd/engine.cpp

```diff
@@ -50,6 +51,8 @@ namespace torch {
 namespace autograd {
 
 namespace {
+static Engine::compiled_autograd_fn the_compiled_autograd = nullptr;
```

Comment:
This static gives internal linkage but scoped to the namespace. Intentional? or did you want a global static?

Reply:
Yes this was intentional.